altlas/Assets/scripts/MapGenerator.cs

```diff
@@ -1,65 +1,348 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 public class MapGenerator : MonoBehaviour {
 
     public Transform map;
-    private float DRAWER_HOR_LEN = 0.36f;
-    private float DRAWER_VER_LEN = 0.44f;
+    private Loader loader;
+    private bool once = true;
 
-    List<MapClass> allMaps = new List<MapClass>();
+    //SCENE CONSTANTS FOR PLACING
+    private float DRAWER_HOR_LEN = 0.12f;
```

Comment:
We should have a look into not using hard coded coordinates

Reply:
Are Ver and Hor short for vertical and horizontal?

Reply:
yes